infra/sandboxes/computer/control.py

```
@@ -0,0 +1,270 @@
+    title = output(["xdotool", "getwindowname", window]) if window else ""
+    source = native_capture(display)
+    if source:
+        try:
+            encoded, width, height, damage = source.copy()
+            image = subprocess.CompletedProcess([], 0, encoded, b"")
+        except RuntimeError:
+            drop_native_capture(display)
+            source = None
+    if not source:
+        image = subprocess.run(
+            ["import", "-define", "png:compression-level=3", "-window", "root", "png:-"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)